server/routes/appointments/occurrence-details/handlers/occurrenceMovementSlip.ts

```diff
@@ -1,9 +1,20 @@
 import { Request, Response } from 'express'
+import { trackEvent } from '../../../../utils/eventTrackingAppInsights'
 
 export default class OccurrenceMovementSlipRoutes {
   GET = async (req: Request, res: Response): Promise<void> => {
     const { appointmentOccurrence } = req
 
     res.render('pages/appointments/movement-slip/occurrence', { appointmentOccurrence })
+
+    const properties = {
+      username: res.locals.user.username,
+      prisonCode: res.locals.user.activeCaseLoadId,
+      appointmentId: appointmentOccurrence.appointmentId.toString(),
+    }
+    const metrics = {
+      appointmentCount: 1.0,
```

Comment:
Again, this is a inaccurate requirement on the ticket. Please could you rename this property to `movementSlipCount` and use `appointmentOccurrence.prisoners.length` for it's value

Reply:
Done